### PR TITLE
Fix connector with latest ballerina changes

### PR DESCRIPTION
**Pull request by Biruntha**
## Purpose
Fix connector with latest ballerina changes.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: jira7/jira_endpoint.bal [hunk 1]
```diff
@@ -18,35 +18,32 @@
 
 import ballerina/http;
 
-documentation{ Represents the Jira Client Connector Endpoint configuration.
-    F{{clientConfig}} Http client endpoint configuration
-}
+# Represents the Jira Client Connector Endpoint configuration.
+# + clientConfig - Http client endpoint configuration
```
Comment (pubudu91) [inline jira7/jira_endpoint.bal]: Http -> HTTP
Fix in all places

File: jira7/jira_endpoint.bal [hunk 2]
```diff
@@ -18,35 +18,32 @@
...
 };
 
-documentation{ Represents the Jira Client Connector Endpoint object.
-    F{{jiraConfig}} jira client Connector endpoint configuration
-    F{{jiraConnector}} jira client connector object
-}
+# Represents the Jira Client Connector Endpoint object.
+# + jiraConfig - jira client Connector endpoint configuration
+# + jiraConnector - jira client connector object
+# + clientConfig - jira client config
```
Comment (pubudu91) [inline jira7/jira_endpoint.bal]: Inconsistent use of cases. If we are referring to the connector in its full name, refer to it as Jira Client Connector. Check all the places

File: jira7/jira_types.bal [hunk 3]
```diff
@@ -35,20 +34,19 @@ public type ProjectSummary record {
...
+# + key - project key
+# + name - project name
+# + description - project description
+# + leadName - jira username of the project lead
+# + projectTypeKey - type of the project("software" or "business")
+# + avatarUrls - project avatar urls
+# + projectCategory - details of project category
+# + issueTypes - support issue types of the project
+# + components - summarized details about components of the project
+# + versions - detatils of project versions
```
Comment (pubudu91) [inline jira7/jira_types.bal]: Again, inconsisten use of cases. jira -> Jira, url -> URL
And sentence case should be used (i.e., first letter upper case)

**Comment by copenat** [comment]
Will a new version be pushed to Ballerina Central?

File: jira7/jira_connector.bal [hunk 4]
```diff
@@ -211,13 +205,11 @@ function JiraConnector::createProject(ProjectRequest newProject) returns Project
...
 
-documentation{Updates a project. Only non null values sent in 'ProjectRequest' structure will
-    be updated in the project. Values available for the assigneeType field are: 'PROJECT_LEAD' and 'UNASSIGNED'.
-    P{{projectIdOrKey}} unique string which represents the project id or project key of a jira project
-    P{{update}} record which contain fields which need to be updated
-    R{{^"boolean"}} returns true if the process is successful
-    R{{JiraConnectorError}} 'JiraConnectorError' record
-}
+# Updates a project. Only non null values sent in `ProjectRequest` structure will
+#    be updated in the project. Values available for the assigneeType field are: `PROJECT_LEAD` and `UNASSIGNED`.
```
Comment (Shan1024) [inline jira7/jira_connector.bal]: Formatting issue?

**Pull request merged**